fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -1187,6 +1189,10 @@ public boolean commitAndPublishTransaction(DatabaseIf db, List<Table> tableList,
         beforeCommitTransaction(tableList, transactionId, timeoutMillis);
         try {
             commitTransaction(db.getId(), tableList, transactionId, tabletCommitInfos, txnCommitAttachment);
+        } catch (Exception e) {
+            LOG.info("release delete bitmap lock,commit txn=" + transactionId + ",catch exception=" + e.getMessage());
+            removeDeleteBitmapUpdateLock(tableList, transactionId);
```

Comment:
you'd better to call `getDeleteBitmapUpdateLock` and `removeDeleteBitmapUpdateLock` in the same function.